templates/users/cancel_deletion.html

```diff
@@ -0,0 +1,20 @@
+{% extends "users/profile_base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+  <div class="container">
+    <h1>{% trans "Cancel scheduled account deletion" %}</h3>
```

Comment:
`<h1>...</h3>`

perhaps h2 would be a good compromise

Reply:
lol oops